Files/Filesystem/StorageFileHelpers/StorageFileExtensions.cs

```diff
@@ -217,6 +217,11 @@ public async static Task<IList<StorageFileWithPath>> GetFilesWithPathAsync(this
 
         public async static Task<IList<StorageFolderWithPath>> GetFoldersWithPathAsync(this StorageFolderWithPath parentFolder, string nameFilter, uint maxNumberOfItems = uint.MaxValue)
         {
+            if (parentFolder == null)
```

Comment:
I'd revert this change. There is nothing particularly evil about that exception. The exception in related to the address bar suggestions, not really to the addressbar text and is used to show the "No suggestions" message below the box. There are nicer ways to do this but the exception does its job.

Reply:
I can revert it if you want, but I didn't see any difference on the adress bar suggestions after this change. The "no suggestions" is still working as expected. The only change I saw is that every time I write a letter on the address bar debugging won't get stopped by that exception.

I will test now your added commit.

Reply:
It works because a different exception (ArgumentNullException) is thrown on the next line :)  Anyway on second thought I this is a fine change 👍

Reply:
Aaah, okay!

Thanks, I appreciate it, it was very annoying for me that exception 😢